package.json

```diff
@@ -30,7 +30,9 @@
     "node": ">=4"
   },
   "dependencies": {
-    "lodash": "^4.17.4",
+    "lodash.isnumber": "^3.0.3",
+    "lodash.isobject": "^3.0.2",
+    "lodash.isstring": "^4.0.1",
```

Comment:
Modular packages are deprecated as well, you can do this instead:

- `isNumber(value)`: `typeof value === 'number'`
- `isString(value)`: `typeof value === 'string'`
- `isObject(value)`: `value !== null && typeof value === 'object'`

This is not strictly equivalent but should be enough :smiley:

Reply:
I've got some proven implementation from various packages, that also handle some ambiguous cases, and decided to move them in separate module. Will that be better?